cacheops/conf.py

```diff
@@ -23,6 +23,7 @@ class Settings(object):
     CACHEOPS_CLEANUP_BATCH_SIZE = 1000
     FILE_CACHE_DIR = '/tmp/cacheops_file_cache'
     FILE_CACHE_TIMEOUT = 60*60*24*30
+    CACHEOPS_LONG_DISJUNCTION = 30
```

Comment:
This makes the number of items in an `IN` list that will still be included in DNFs configurable.